Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:Carousel" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:Carousel">
+                    <Grid>
+                        <controls:CarouselPanel x:Name="TPanel">
+
+                        </controls:CarouselPanel>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="controls:CarouselItem" >
```

Comment:
you don't need this, you can remove it
<Style TargetType="controls:CarouselItem" >
        <Setter Property="Template">
            <Setter.Value>
                <ControlTemplate TargetType="controls:CarouselItem">
                    <ContentPresenter />
                </ControlTemplate>
            </Setter.Value>
        </Setter>
    </Style>